site/src/pages/author.js

```diff
@@ -1,144 +1,174 @@
+
+  return (
+    <>
+      <header>
+        <h1>{data.author.elements.name.value}</h1>
+      </header>
+      <section>
+        <ImageElement
+          alt={avatar.description}
+          {...imageData}
+        />
+        <p>
+          This uses the <a href="https://github.com/kontent-ai/gatsby-packages/tree/master/packages/gatsby-components">ImageElement</a> component to display a
+          responsive image.
+        </p>
+
+        <div style={{ "max-width": "800px", "overflow": "auto" }}>
+          <p>
+            There is also exported function called getGatsbyImageData to obtain data used for GatsbyImage specifically. The following example shows the data for the image above.
+          </p>
+          <pre>{`const imageData = getGatsbyImageData({ image: avatar, width: 800, height: 200, backgroundColor:"#bbbbbb"})`}
+          </pre>
+          <pre>
+            {JSON.stringify(imageGatsbyData, undefined, 2).replace(/\\n/g, "\n")}
+          </pre>
+        </div>
+      </section>
+
+      <article
+        style={{ display: "flex", flexWrap: "wrap", alignItems: "flex-end" }}
+      >
+        <section style={{ padding: "1em", maxWidth: showcaseWidth }}>
+          <img
+            src={avatar.url}
+            alt={avatar.description || "Author avatar image"}
+            width="200px"
+          />
+          <p>
+            This is the basic loading of the asset in full size and setting its
+            width right in <tt>img</tt> tag (don't do that).
+          </p>
+        </section>
+        <section style={{ padding: "1em", maxWidth: showcaseWidth }}>
+          <img
+            src={`${avatar.url}?${stringify(imageQuery, { encode: false })}`}
+            alt={avatar.description || "Author avatar image"}
+          />
+          <p>
+            Showcase using{" "}
+            <a href="https://docs.kontent.ai/reference/image-transformation">
+              Kontent.ai Image transformation API
+            </a>{" "}
+            and <a href="https://www.npmjs.com/package/qs">qs npm package.</a>{" "}
+            for query string construction.
+          </p>
+        </section>
+      </article>
+
+      <article
+        style={{ display: "flex", flexWrap: "wrap", alignItems: "flex-end" }}
+      >
+        <section style={{ padding: "1em", maxWidth: showcaseWidth }}>
+          <img
+            src={deliverySDKTransformedUrl.getUrl()}
+            alt={avatar.description || "Author avatar image"}
+          />
+          <p>
+            Showcase using{" "}
+            <a href="https://docs.kontent.ai/reference/image-transformation">
+              Kontent.ai Image transformation API
+            </a>{" "}
+            and{" "}
+            <a href="https://www.npmjs.com/package/@kontent-ai/delivery-sdk">
+              Delivery SDK
+            </a>{" "}
+            for query string construction.
+          </p>
+        </section>
+
+
+
+        {avatar.renditions &&
```

Comment:
Suggestion: I would showcase this in readme as renditions might not be there and explain why (also renditions are not allowed in developer plan) or atleast add comment here to explain why they should not forget check if renditions are there. What do you think?